blocks/feed/feed.js

```diff
@@ -76,14 +76,24 @@ const resultParsers = {
         }
       });
       if (cardImage) {
-        row.push(cardImage);
+        if (result.path) {
+          const pathImg = document.createElement('a');
+          pathImg.href = result.path;
+          pathImg.append(cardImage);
+          row.push(pathImg);
+        } else row.push(cardImage);
+      }
+      if (meta === 'featured') {
```

Comment:
Rather than deriving this from `meta` , it should be derived from the `result` . Basically show `FEATURED` if its a featured article.